app/src/main/java/org/oppia/app/player/state/itemviewmodel/ContentViewModel.kt

```diff
@@ -1,5 +1,14 @@
 package org.oppia.app.player.state.itemviewmodel
 
 /** [StateItemViewModel] for content-card state. */
-class ContentViewModel(val htmlContent: CharSequence, val gcsEntityId: String) :
-  StateItemViewModel(ViewType.CONTENT)
+class ContentViewModel(
+  val htmlContent: CharSequence,
+  val gcsEntityId: String,
+  hasConversationalContentView: Boolean
+) : StateItemViewModel(ViewType.CONTENT) {
+  var hasConversationalContentView: Boolean = false
+
+  init {
+    this.hasConversationalContentView = hasConversationalContentView
+  }
+}
```

Comment:
This can be simplified to this:
```
/** [StateItemViewModel] for content-card state. */
class ContentViewModel(
  val htmlContent: CharSequence,
  val gcsEntityId: String,
  val hasConversationalContentView: Boolean
) : StateItemViewModel(ViewType.CONTENT)

```

Reply:
Done.

Reply:
updated